src/Components/CreateImageWizardV2/EditImageWizard.tsx

```diff
@@ -36,7 +36,7 @@ const EditImageWizard = ({ blueprintId }: EditImageWizardProps) => {
       navigate(resolveRelPath(''));
     }
   }, [error, navigate]);
-  return <CreateImageWizard startStepIndex={13} />;
+  return <CreateImageWizard startStepIndex={14} />;
```

Comment:
Will probably need to bump this to 15 if we get the Snapshotting PR in first :thinking: